games-arcade/hikounomizu/hikounomizu-1.1.recipe

```diff
@@ -1,15 +1,16 @@
 SUMMARY="A platform-based, anime-styled fighting game"
 DESCRIPTION="Hikou no mizu is a 2D fighting game with anime-inspired \
-graphics. Fights take place in interactive arenas of various sizes and can \
-involve up to five players on a single machine. Currently, two players are \
-playable, namely Hikou and Takino. Joysticks are supported and a simple AI \
-is included for solo playing."
+graphics. Fights take place in interactive arenas of various sizes. Both \
+networked multiplayer and local games are possible. Joysticks are \
+supported and a simple AI is included for solo playing. Playable \
+characters fight using natural powers such as water, lightning or fire. \
+Currently, three characters are playable: Hikou, Takino and Hana."
```

Comment:
Since you are sorting DESCRIPTION, lines can go up to 100 characters (I'm guessing you kept the 80 char limit).